packages/integrations/src/debug.ts

```diff
@@ -49,12 +49,12 @@ export class Debug implements Integration {
         consoleSandbox(() => {
           if (self._options.stringify) {
             console.log(JSON.stringify(event, null, 2));
-            if (hint) {
+            if (Object.keys(hint).length) {
```

Comment:
Fixing these types to actually represent what is passed has highlighted  that this would have always been logging an empty object for an empty hint.